array-manipulation-2/src/drop-right.js

```diff
@@ -0,0 +1,11 @@
+/* exported dropRight */
+function dropRight(array, count) {
+  var takeArray = [];
+  for (var i = 0; i < array.length; i++) {
+    var newValue = (array.length - count);
+    if (i < newValue) {
+      takeArray.push(array[i]);
+    }
```

Comment:
You can simplify this a bit by setting `i < (array.length - count)` as your loop's condition expression.